lib/cf_deployer/driver/cloud_formation_driver.rb

```diff
@@ -21,13 +21,18 @@ def update_stack template, opts
           CfDeployer::Driver::DryRun.guard "Skipping update_stack" do
             aws_stack.update opts.merge(:template => template)
           end
+
+          return !CfDeployer::Driver::DryRun.enabled?
         rescue AWS::CloudFormation::Errors::ValidationError => e
           if e.message =~ /No updates are to be performed/
             Log.info e.message
+            return false
           else
             raise
           end
         end
+
+        true
```

Comment:
Maybe it would be better written like this?  

https://gist.github.com/kmanning/0fe2e9fe08cad72066999748d27b4a97